CMakeLists.txt

```diff
@@ -157,6 +157,14 @@ set_option_if(ACTS_BUILD_PLUGIN_LEGACY ACTS_BUILD_EVERYTHING)
 set_option_if(ACTS_BUILD_PLUGIN_EXATRKX ACTS_BUILD_EXAMPLES_EXATRKX)
 set_option_if(ACTS_BUILD_PLUGIN_FPEMON
   ACTS_BUILD_EXAMPLES OR ACTS_BUILD_EVERYTHING)
+set_option_if(ACTS_SETUP_ALGEBRAPLUGINS
+  ACTS_BUILD_EXAMPLES OR ACTS_BUILD_EVERYTHING)
+set_option_if(ACTS_SETUP_VECMEM
+  ACTS_BUILD_EXAMPLES OR ACTS_BUILD_EVERYTHING)
```

Comment:
Are these changes related to this PR?

Reply:
I talked with Paul about the cmake set up. Now setup is enabled by default for vecmem, algebra plugins, covfie, and detray, and the covfie plugin is included in the traccc plugin build option.